src/components/application_manager/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -359,6 +372,92 @@ TEST_F(RegisterAppInterfaceRequestTest,
   command_->Run();
 }
 
+MATCHER_P(CheckMobileResponseParameters, ui_hmi_capabilities, "") {
+  const bool is_result_id_correct =
+      mobile_apis::Result::SUCCESS ==
+      static_cast<mobile_apis::Result::eType>(
+          (*arg)[am::strings::msg_params][am::strings::result_code].asInt());
+  const bool is_navigation_value_valid =
+      (*arg)[am::strings::msg_params][am::strings::hmi_capabilities]
+            [am::strings::navigation].asBool() ==
+      (*ui_hmi_capabilities)[am::strings::navigation].asBool();
+  const bool is_phone_call_value_valid =
+      (*arg)[am::strings::msg_params][am::strings::hmi_capabilities]
+            [am::strings::phone_call].asBool() ==
+      (*ui_hmi_capabilities)[am::strings::phone_call].asBool();
+  const bool is_steering_wheel_location_value_valid =
+      (*arg)[am::strings::msg_params][am::strings::hmi_capabilities]
+            [am::strings::steering_wheel_location].asInt() ==
+      (*ui_hmi_capabilities)[am::strings::steering_wheel_location].asInt();
+  return is_result_id_correct && is_navigation_value_valid &&
+         is_phone_call_value_valid && is_steering_wheel_location_value_valid;
```

Comment:
@LevchenkoS , think that making some fixes is out scope of this task, because time was estimated only for creating separate PR